Test/UnitTest1.cs

```diff
@@ -0,0 +1,46 @@
+using static Spark23.T27;
+
+namespace Test;
+
+[TestClass]
+public class UnitTest1 {
```

Comment:
Even though the coverage is 100%, the tests are not so effective.
Use random numbers to push into the stack and it should be like 4 pushes 2 pops and then 2 pushes and a pop etc.

Also, it has been advised in the workshop to use inbuilt Stack to test the data of your TStack. So, you have to push/pop in both TStack and Stack and verify if the Pop of both are same. If you do this then you don't need that Size property in your TStack class.